manifests/mod/passenger.pp

```diff
@@ -1,5 +1,7 @@
 # @summary
 #   Installs `mod_pasenger`.
+#   > **Note**: This module does not install the passenger gem/package. You must install it yourself.
```

Comment:
I'd assume that if you pass `mod_package` that it does.

Reply:
you are right, its does. Looks I missed that part totally. Thanks for catching this.